x-pack/test/apm_api_integration/tests/dependencies/top_spans.spec.ts

```diff
@@ -223,34 +223,6 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         });
       });
 
-      describe('when requesting spans without a transaction', () => {
```

Comment:
I've removed this test, as showing a span where we can't find a transaction for is IMHO pointless, the span won't show up in the waterfall.

Reply:
Do you recall why you added this test in the first place? It was added in [this PR](https://github.com/elastic/kibana/pull/134179)

Reply:
Btw. Perhaps add an e2e test that selects three different traces and verifies that the trace view updates accordingly.

Reply:
@sqren there was no requirement to show traces then, so I thought it would be good to take it into account. But I don't think it made much sense before either, it only covers a hypothetical scenario where a transaction is lost but the span is still ingested. I've tried to stay away from E2E testing and rather add a unit test because I'm skeptical of the stability of the E2E test considering the redirects that take place, but I can add one if you feel it's necessary?

Reply:
Feel free to skip e2e. I'll leave you decide the most feasible way to get adequate coverage.